schemas/real-time-data-feed.schema.json

```
@@ -0,0 +1,65 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://region-registry.skedgo.com/real-time-data-feed.schema.json",
+  "title": "Real Time Data Feed",
+  "description": "Real Time Data Feed specification",
+  "type": "object",
+  "properties": {
+    "id": {
+      "description": "Unique identifier for this feed. Recommended format is ISO Country Code + Provider name; for example au-tfnsw. For multi country feeds, recommended format is Continent Code + Provider name; for example eu-flixbus.",
+      "type": "string",
+      "pattern": "^[a-z]{2}-(([a-z]|-|_)+)$"
+    },
+    "name": {
+      "description": "Human friendly name for this feed.",
+      "type": "string",
+      "minLength": 1,
+      "pattern": "^(([A-Z]|[a-z]|-|_)+)$"
```

Why doesn't this allow spaces or numbers? Why limit it at all for that matter?

```suggestion
      "minLength": 1
```